src/plugins/event-plugins/FetchPlugin.ts

```diff
@@ -272,7 +289,11 @@ export class FetchPlugin extends MonkeyPatched<Window, 'fetch'> {
             return original.apply(thisArg, argsArray as any);
         }
 
-        if (this.isTracingEnabled() && this.isSessionRecorded()) {
+        if (
+            !isSyntheticsUA() &&
```

Comment:
suggestion: Look for an existing trace header before we reach this point. If there is a header, then simply set `httpEvent.trace_id` and `httpEvent.segment_id`. If there is no header, then `beginTrace(...)`

I don't think we need to use `isSyntheticsUA` here, since we know whether or not there is a trace header.